executor/platform_compat/windows/user_info.py

```
@@ -0,0 +1,120 @@
+        Returns:
+            Current username or UID as string.
+        """
+        try:
+            return os.getlogin()
+        except Exception:
+            try:
+                return os.environ.get("USERNAME", str(uid))
+            except Exception:
+                return str(uid)
+
+    def get_group_name(self, gid: int) -> str:
```

Ruff (0.14.14)

[warning] Unused method argument: `gid`

(ARG002)